src/v/config/configuration.cc

```diff
@@ -1609,7 +1609,21 @@ configuration::configuration()
       "balancer, in milliseconds",
       {.needs_restart = needs_restart::no, .visibility = visibility::user},
       5000ms,
-      {.min = 1ms, .max = bottomless_token_bucket::max_width}) {}
+      {.min = 1ms, .max = bottomless_token_bucket::max_width})
+  , node_isolation_raft_timeout(
+      *this,
+      "health_monitor_append_entries_timeout_ms",
+      "Max time to waiting append_entries request before be isoalted",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      3000,
+      {.min = 100, .max = 10000})
+  , node_isolation_heartbeat_timeout(
+      *this,
+      "node_isolation_heartbeat_timeout",
+      "Max time to waiting out of raft heartbeats before be isoalted",
```

Comment:
Please see John's comment on the description, this is user facing part and we need to be very clear on what that property means